client/src/App.js

```diff
@@ -4,6 +4,8 @@ import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
 
+import {Route} from 'react-router-dom';
```

Comment:
Perfect, do you feel comfortable when to use {var} from ... and when to use var from ... ?